docker/Dockerfile

```diff
@@ -0,0 +1,44 @@
+FROM debian:bullseye-slim
```

Comment:
could we pick a smaller base image? it's go app, it's cross-compiled over many architectures here, not quite sure why do we need a docker image for this

Reply:
Do you have a suggestion? After building, using bullseye-slim, the image is about 120MB. With the official golang image would be over 1GB.

Reply:
Will scratch work?..